src/Setup.tsx

```diff
@@ -86,28 +88,30 @@ export class Setup extends PureComponent<Props, State> {
     const {loading, allowed} = this.state
 
     return (
-      <PageSpinner loading={loading}>
-        <Suspense fallback={<PageSpinner />}>
-          {allowed && (
-            <Route
-              path="/onboarding/:stepID"
-              component={OnboardingWizardPage}
-            />
-          )}
-          {!allowed && (
-            <Switch>
+      <GetLinks>
```

Comment:
This move was done since the parent component is rendering a switch that was previously always rendering the GetLinks and Setup routes